evm_arithmetization/tests/add11_yml.rs

```diff
@@ -170,19 +170,24 @@ fn add11_yml() -> anyhow::Result<()> {
     let config = StarkConfig::standard_fast_config();
     let inputs = get_generation_inputs();
 
-    let mut timing = TimingTree::new("prove", log::Level::Debug);
     let max_cpu_len_log = 20;
-    let segment_idx = 0;
+    let mut data = generate_all_data_segments::<F>(Some(max_cpu_len_log), inputs.clone())?;
+
+    assert_eq!(data.len(), 2);
```

Comment:
So we have a single segment, but 2 `GenerationSegmentData` to accomodate for the `registers_after` of the first and only segment? We may want to have it emphasized somewhere, I got slightly confused at first.

Reply:
I added a comment in `generate_all_data_segments`. I also updated the method to have all `GenerationSegmentData` values except the registers be set to default values for the final segment, to save at least some memory.